src/lib/mockserver.js

```diff
@@ -139,6 +118,47 @@ class MockServer {
             }
         }
     }
+
+    /**
+     * Spawn docker logs until mocks are set up
+     */
+    async _dockerSpawnLogs() {
+        debug(`> docker logs -f ${this.container}`);
+        const proc = spawn('docker', ['logs', '-f', this.container]);
+        let isRunning;
+
+        proc.stdout.on('data', function(data) {
+            const stdout = data.toString();
+            if (stdout && stdout.includes('started on ports: [80, 443]')) {
+                debug("Stdout", stdout);
+                isRunning = true;
+            }
+        });
+        proc.stderr.on('data', function(data) {
+            process.stderr.write(data.toString());
+        });
```

Comment:
I think you don't need that at all if the desired output ("started on ports") is on `stdout`.